helm/superset/values.yaml

```
@@ -165,6 +165,25 @@ supersetWorker:
             name: '{{ tpl .Values.envFromSecret . }}'
       command: [ "/bin/sh", "-c", "until nc -zv $DB_HOST $DB_PORT -w1; do echo 'waiting for db'; sleep 1; done" ]
 
+##
+## Superset beat configuration (to trigger scheduled jobs like reports)
+supersetBeat:
```

We also renamed this one to be consistent
```suggestion
supersetCeleryBeat:
```